data/data/openstack/main.tf

```diff
@@ -72,3 +72,21 @@ module "topology" {
   trunk_support       = var.openstack_trunk_support
   octavia_support     = var.openstack_octavia_support
 }
+
+resource "openstack_images_image_v2" "base_image" {
+  // we need to create a new image only if the base image url has been provided + base image name is <cluster_id>-rhcos
+  count = var.openstack_base_image_url != "" && var.openstack_base_image == "${var.cluster_id}-rhcos"? 1 : 0
```

Comment:
Thanks.
`terraform fmt` wants to add a space before the question mark.

Reply:
yep, done